extensions/amp-carousel/0.1/base-carousel.js

```diff
@@ -60,24 +60,20 @@ export class BaseCarousel extends AMP.BaseElement {
 
   /**
    * Handles element specific viewport based events.
-   * @param {boolean} unusedInViewport.
+   * @param {boolean} unusedInViewport
    * @protected
    */
   onViewportCallback(unusedInViewport) {}
 
 
+  /**
+   * Builds the next and previous buttons.
+   */
   buildButtons() {
```

Comment:
Nit: not that you caused this, but a large part of the code between building the prev and next buttons are duplicate. We could refactor this to reduce quite a bit of duplicate code. =)

Reply:
Combined the logic. Seems like a bit of a trade-off for being straightforward vs. adding some indirection, but doesn't seem too bad here.